epick_description/urdf/epick_macro.xacro

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="robotiq_epick"
+               params="prefix
+                       parent
+                       *origin
+                       extension_radius:=0.0
+                       extension_length:=0.0
+                       suction_cup_radius:=0.012
```

Comment:
These default values are somewhat arbitrary. Maybe they should be 0 so that users would have to configure their own values.

Reply:
I think providing reasonable defaults based on what Robotiq recommends / ships with + a comment telling users to change it is a good start

Reply:
I have not found actual specs for what cups they ship with the epick. I wouldn't be surprised if those are not always the same, so people might have to figure out what they are using.